### PR TITLE
[modules] Updating analyze.sh to check for all forced modules

### PR DESCRIPTION
**Pull request by brianjjones**
There were a couple modules that got left off, such as Buffer and
Timers.  Now these can be built adding them to the config file.

### CODE REVIEW

File: scripts/analyze.sh [hunk 1]
```diff
@@ -25,7 +25,7 @@ BOARD=$1
 SCRIPT=$2
 CONFIG=$3
 
-echo "# Modules found in $SCRIPT:" > prj.conf.tmp
+echo "# Modules found in $SCRIPT:" > prj.conf.tmpq
```
Comment (grgustaf) [inline scripts/analyze.sh]: typo?

File: scripts/analyze.sh [hunk 2]
```diff
@@ -189,27 +200,32 @@ if check_for_require arduino101_pins || check_config_file ZJS_ARDUINO101_PINS; t
...
 else
     timeout=$(grep setTimeout $SCRIPT)
     if [ $? -eq 0 ]; then
         MODULES+=" -DBUILD_MODULE_TIMER"
+        echo "export ZJS_TIMERS=y" >> zjs.conf.tmp
     else
         immediate=$(grep setImmediate $SCRIPT)
         if [ $? -eq 0 ]; then
             MODULES+=" -DBUILD_MODULE_TIMER"
+            echo "export ZJS_TIMERS=y" >> zjs.conf.tmp
```
Comment (grgustaf) [inline scripts/analyze.sh]: These two lines are repeated 3x, I think they could be here once if you restructured the if statements to check the reverse like if !0 && ! check_config && !timeout && !immediate, then timers=false. Default, timers=true. But not a big deal.

File: scripts/analyze.sh [hunk 3]
```diff
@@ -189,27 +200,32 @@ if check_for_require arduino101_pins || check_config_file ZJS_ARDUINO101_PINS; t
...
         fi
     fi
 fi
-buffer=$(grep Buffer\([0-9]*\) $SCRIPT)
-if [ $? -eq 0 ] && [[ $MODULE != *"BUILD_MODULE_BUFFER"* ]]; then
+
+
```
Comment (grgustaf) [inline scripts/analyze.sh]: extra blank line here

File: scripts/analyze.sh (hunk 2 above)
Comment (brianjjones) [inline scripts/analyze.sh]: Even shorter now, should do the trick

**Comment by grgustaf** [comment]
+1

**Comment by jimmy-huang** [comment]
+1

**Comment by jimmy-huang** [comment]
@brianjjones can you update commit with signed off by?

**Pull request merged**